controllers/example_controller/keyboard_controller.py

```diff
@@ -1,7 +1,7 @@
 from sr.robot import Robot
 from controller import Keyboard
 
-KEYBOARD_SAMPLING_FREQUENCY = 16
+KEYBOARD_SAMPLING_PERIOD = 100
```

Comment:
Maybe worth a comment about how to choose this value now that it's substantially different than the timestep?